amp/TEMPLATE/reampv2/src/modules/sscdashboard/components/layout/filters/horizontal-filters.jsx

```diff
@@ -158,6 +166,13 @@ class HorizontalFilters extends Component {
 
         }
     }
+    // TODO to be removed
```

Comment:
Is this still valid?

Reply:
yes is still valid since until we have the data coming from the endpoint I need it to generate mock data. Ill remove it after committing next ticket